### PR TITLE
Modified `add` verb to support app IDs

### PR DESCRIPTION
**Pull request by eBardX**

### CODE REVIEW

File: waldo/add.go [hunk 1]
```diff
@@ -104,35 +114,62 @@ func (aa *AddAction) confirmRecipe(basePath string, recipe *data.Recipe) bool {
...
+
+	if err != nil {
+		return "", "", err
+	}
+
+	if platform != lib.PlatformUnknown {
+		items = lib.CompactMap(items, func(item *tool.AppInfo) (*tool.AppInfo, bool) {
+			return item, item.Platform == platform
+		})
+	}
```
Comment (VincentJaouen) [inline waldo/add.go:162]: Shouldn't this already be done by the backend if the query param is set in the request?

**Review by VincentJaouen — APPROVED** [review]
LGTM

File: waldo/add.go (hunk 1 above)
Comment (eBardX) [reply waldo/add.go:162]: Yes, but I believe in belt and suspenders.

File: waldo/upload.go [hunk 2]
```diff
@@ -200,9 +200,9 @@ func (ua *UploadAction) makeArgs(buildPath, uploadToken string) []string {
 func (ua *UploadAction) mungeArgs(ud *data.UserData, recipe *data.Recipe) ([]string, string, error) {
 	uploadToken := ua.options.UploadToken
 
-	if len(uploadToken) == 0 {
-		uploadToken = recipe.UploadToken
-	}
+	// if len(uploadToken) == 0 {
+	// 	uploadToken = recipe.UploadToken
+	// }
```
Comment (vbfox) [inline waldo/upload.go:205]: Commented code

**Pull request merged**